tests/conftest.py

```
@@ -512,6 +527,41 @@ def mock_submit(coro, loop=None):
     yield
 
 
+def _fast_submit(coro, loop=None):
```

Ruff (0.14.8)

Unused function argument: `loop`

(ARG001)